.env.template

```
@@ -1,6 +1,8 @@
 GF_SECURITY_ADMIN_USER="cks" # username for logging into Grafana
 GF_SECURITY_ADMIN_PASSWORD="pass" # password for logging into Grafana
 BASIC_AUTH_PASSWORD="pass" # password for basic auth used by Prometheus and Pushgateway
+BASIC_AUTH_WEBSITE_USERNAME="preview" # username for basic auth used in our websites
+BASIC_AUTH_WEBSITE_PASSWORD="123NewCKEditorSite" # password for basic auth used in our websites
```

Let's hide the actual password from github files

```suggestion
BASIC_AUTH_WEBSITE_PASSWORD="" # password for basic auth used in our websites
```